components/match_ticker/commons/match_ticker_display_components.lua

```diff
@@ -311,7 +311,7 @@ function Details:countdown(matchPageIcon)
 
 		local gameVods = Array.map(Array.map(match.match2games, Operator.property('vod')), makeVod)
 
-		countdownDisplay:node(makeVod(match.vod))
+		countdownDisplay:node(makeVod(match.links.vod))
```

Comment:
need to adjust `DEFAULT_QUERY_COLUMNS` too
and check if there are places overwriting that default somewhere

and also infobox player on sc2)